README.md

```diff
@@ -1 +1,8 @@
-# behat-transformation-context
\ No newline at end of file
+# Behat Transformation Context
+
+[![Build Status](https://drone.fpfis.eu/api/badges/openeuropa/behat-transformation-context/status.svg)](https://drone.fpfis.eu/openeuropa/behat-transformation-context/)
+
+This package makes available a Context allowing to transform human readable labels to selectors or page paths.
```

Comment:
The phrasing is a bit weird, I think it would be best to say "This package provides a Behat context...""